src/Models/ForeignRelationship.php

```diff
@@ -29,6 +29,7 @@ class ForeignRelationship implements JsonWriter
         'morphMany',
         'morphOne',
         'morphToMany',
+        'morphedByMany'
```

Comment:
Unlike the monomorphic case, where the other end of a belongsToMany is itself a belongsToMany, a polymorphic many-to-many relation is sensitive to direction.  Thus, the other end of a morphToMany is NOT another morphToMany, but a morphedByMany.